### ISSUE TITLE
Error executing the example code

### ISSUE THREAD

**Issue by kiey** [DiaKEM/libre-link-up-api-client#16]
Hello,
Sorry because I am new to JS and maybe this question is due to that.

I tried the snipped of code, having the plain username and the password.
```
import {LibreLinkUpClient} from '@diakem/libre-link-up-api-client';


const {read} = LibreLinkUpClient({username: ' ', password: ' '});

const response = await read();
```

And I get:
home/joan/node_modules/@diakem/libre-link-up-api-client/lib/client.js:73
              if (!loginResponse.data.data.redirect) {
                                           ^

TypeError: Cannot read properties of undefined (reading 'redirect')
    at _callee$ (/home/joan/node_modules/@diakem/libre-link-up-api-client/lib/client.js:73:44)
    at tryCatch (/home/joan/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/joan/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/joan/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/joan/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/joan/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

To execute, I saved the code in a `test.mjs` file and run with `node test.mjs`

**Comment by NzoSifou** [comment]
It's because you don't have valid username and password.
I tested with good credentials and I got a different error:

`C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@diakem\libre-link-up-api-client\src\client.ts:147
    return connections[0].patientId;
                          ^
TypeError: Cannot read properties of undefined (reading 'patientId')
    at getConnection (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@diakem\libre-link-up-api-client\src\client.ts:147:27)
    at _callee4$ (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@diakem\libre-link-up-api-client\src\client.ts:154:22)
    at tryCatch (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\nzosi\Desktop\Nouveau dossier\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)`

**Comment by kiey** [comment]
Thanks for your answer.
You were right, I was mistaken and I was using the credentials from librelinkup,
With the right credentials I get a different error than yours.

```
node:internal/process/esm_loader:94
    internalBinding('errors').triggerUncaughtException(
                              ^
[AxiosError: Request failed with status code 401] {
  code: 'ERR_BAD_REQUEST',
  config: {
    transitional: {
      silentJSONParsing: true,
      forcedJSONParsing: true,
      clarifyTimeoutError: false
    },
....



```

**Comment by NzoSifou** [comment]
Hello,
So sorry that's not it, it's my fault, I thought it was for "LibreLink" and not "LibreLinkUp", so I tried with my "LibreLinkUp" ID and it worked.

Your problem is that you have entered bad credentials for your LibreLinkUp account (maybe a typo?), because your first error was because of your credentials anyway.

If it don't work for you, even with good credentials, try to create another account on LibreLinkUp **with the same credentials than your LibreLink account**, because it's what I did and it work perfectly.

Have a nice day!
